Makefile

```diff
@@ -203,6 +203,12 @@ codegen:
 
 ### Docker images ###
 
+.PHONY: base-ubuntu
+base-ubuntu:
```

Comment:
you changed the target name to `base-ubuntu` but not the docker image name, which is what I was suggesting

Reply:
Got it, updated.